modules/core/react/lib/type.ts

```diff
@@ -149,6 +149,7 @@ const variants: CanvasTypeVariant = {
     },
     '&:focus': {
       boxShadow: `0 0 0 2px ${colors.blueberry400}`,
+      outline: 'none',
```

Comment:
no need for outline when there is boxShadow 
<img width="265" alt="Screen Shot 2020-05-27 at 3 36 29 PM" src="https://user-images.githubusercontent.com/10633451/83038927-6cf61980-a035-11ea-9545-bed113dc05e3.png">